minio/mc
# Restructure `mc support callhome` command

## Description

- Remove `support logs enable|disable|status|show`
- Make `support logs` behave like `support logs show`
- Enhance `support callhome` to allow configuration of both periodic diagnostics and real-time log push with the same command
- This will be achieved by adding two bool flags `--diag` and `--logs`
- If none of these flags are passed, the action specified by the arg (enable|disable|status) will be performed for both the features

## Motivation and Context

Group real-time logs and periodic diag push under the same umbrella of `callhome`

## How to test this PR?

- `mc support callhome status {alias}` should print status of both diag and logs
- `mc support callhome enable {alias}` should enable both diag and logs
- `mc support callhome disable {alias}` should disable both diag and logs
- `mc support callhome status {alias} --[diag|logs]` should print status of the given feature (diag/logs)
- `mc support callhome enable {alias} --[diag|logs]` should enable only the given feature (diag/logs)
- `mc support callhome disable {alias} --[diag|logs]` should disable only the given feature (diag/logs)

## Types of changes

## Checklist:
